test/insertjavascript.js

```diff
@@ -0,0 +1,45 @@
+/*globals describe, it, beforeEach */
+'use strict';
+
+var assert = require('assert');
+var GruntfileEditor = require('..');
+var helper = require('./helpers');
+
+describe('#insertJavaScript()', function () {
+
+  beforeEach(function () {
+    this.editor = new GruntfileEditor();
+    this.insert = helper.insertJavaScript;
+    this.str = helper.str;
+  });
+
+  it('is chainable', function () {
+    assert.equal(this.editor.insertJavaScript('require(\'load-grunt-tasks\')(grunt);'), this.editor);
+  });
+
+  describe('requires code', function () {
```

Comment:
no need for this describe block - and no need for a scoped variable. Just make it part of the `it` block.